### PR TITLE
[Prefactoring] Pass brokerClass to Trigger reconciler instead of hardcoding it

### PR DESCRIPTION
**Pull request by aliok**
Fixes #

## Proposed Changes

- Pass brokerClass to Trigger reconciler instead of hardcoding it
- Prefactoring for KafkaBroker with dataplane-per-namespace
-

**Release Note**

```release-note

```

**Docs**

### CODE REVIEW

**Comment by aliok** [comment]
/hold
Blocked by https://github.com/knative-sandbox/eventing-kafka-broker/pull/2360

File: control-plane/pkg/reconciler/trigger/trigger.go [hunk 1]
```diff
@@ -95,7 +96,7 @@ func (r *Reconciler) reconcileKind(ctx context.Context, trigger *eventing.Trigge
 	}
 
 	// Ignore Triggers that are associated with a Broker we don't own.
-	if isKnativeKafkaBroker, brokerClass := isKnativeKafkaBroker(broker); !isKnativeKafkaBroker {
+	if isKnativeKafkaBroker, brokerClass := r.hasRelevantBrokerClass(broker); !isKnativeKafkaBroker {
```
Comment (pierDipi) [inline control-plane/pkg/reconciler/trigger/trigger.go:99]: should we rename the variable `isKnativeKafkaBroker` as well?

**Pull request merged**